guides/common/modules/con_registering-hosts-by-using-global-registration.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Registering_Hosts_{context}"]
```

Comment:
Please change the anchor.

Reply:
See https://github.com/theforeman/foreman-documentation/pull/2348#discussion_r1295393768 for details

Reply:
that is really unfortunate! I think we should look into this again on Monday.